src/main/java/edu/ucsb/cs156/happiercows/entities/UserCommons.java

```diff
@@ -30,6 +30,10 @@ public class UserCommons {
 
   private int numOfCows;
 
+
+  private String username;
```

Comment:
So we did add username to the usercommons in one of the other code bases, but this was a mistake.  We should not have done this, and we are going to have to undo it.

username is already available through the userid, and we should always get it through the user id.  It is bad design to store redundant data in the database.   So this cannot be merged in this form.

Reply:
This link shows how this should be implemented: as additional functionality in the UserCommonsController:

https://github.com/ucsb-cs156-f22/f22-7pm-happycows/blob/main/src/main/java/edu/ucsb/cs156/happiercows/controllers/UserCommonsController.java#L145